src/main/java/uk/ac/ebi/eva/utils/MongoDBHelper.java

```diff
@@ -86,8 +86,4 @@ public static String buildStorageId(String chromosome, int start, String referen
         return builder.toString();
     }
 
-    public static String buildStorageFileId(String studyId, String fileId) {
```

Comment:
I moved that method because has nothing to do with mongo. The underscore is only used as the VariantSourceEntry key in the Variant object and in some files like the stats one: "sid_fid.variants.stats.json"